.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,124 @@
+name: Windows-CI
+on: [push, pull_request]
+
+jobs:
+   windows-build:
+    strategy:
+      matrix:
+        arch: [x64] #[Win32, x64] 
+    runs-on: windows-latest
+    steps:
+      - name: Set Tool-Architecture
+        uses: allenevans/set-env@v1.0.0
+        with:
+          PreferredToolArchitecture: '${{matrix.arch}}'
+          overwrite: true
+      - name: Checkout sources
+        uses: actions/checkout@v1
+      - name: Cache
+        uses: actions/cache@v1.0.2
+        id: cache-python-dl
+        with:
+          path: 'Python38-x64-dbg'
+          key: ${{ runner.OS }}-python
+          restore-keys: |
+            ${{ runner.OS }}-python
+            ${{ runner.OS }}-
```

Comment:
The linkes issues have influence on this part.

Reply:
Someone had a tip how to get parts of the vcpkg-dir cached. Pack them, move them into the `github_workspace` and then cache them from there and vice versa (you can just cache from inside the workspace/cloning dir). I'll test that out later, although I think that the directory will be **>400 MiB** (that is the individual cache size, 2 GiB is the overall cache size) and therefore we maybe need to find another solution. I asked some stuff in their repos about this, so let's see what caching will bring.